### PR TITLE
Ecomm JSON Schema error fixes

### PR DESCRIPTION
**Pull request by nissimsan**
This fixes JSON Schema errors on Ecommerce certificates. These are the ones which are normally hidden, but emerge on the console when other errors exist.

There are three main category of bugs fixed:

- Array const instead of enum, which has been copy-pasted many times
- Changes to Org and Entity propagating to where referenced
- "Normal", one-off JSON Schema bugs (mostly mine)

### CODE REVIEW

**Comment by nissimsan** [comment]
Btw, this also contains a few LD term fixes.

File: docs/openapi/components/schemas/credentials/IATAAirWaybillCertificate.yml [hunk 1]
```diff
@@ -90,66 +86,96 @@ example: |-
       },
       "conditionsOfContract": "It is agreed that the goods described herein are accepted in apparent good order and condition (except as noted) for carriage SUBJECT TO THE CONDITIONS OF CONTRACT ON THE REVERSE HEREOF. ALL GOODS MAY BE CARRIED BY ANY OTHER MEANS INCLUDING ROAD OR ANY OTHER CARRIER UNLESS SPECIFIC CONTRARY INSTRUCTIONS ARE GIVEN HEREON BY THE SHIPPER, AND SHIPPER AGREES THAT THE SHIPMENT MAY BE CARRIED VIA INTERMEDIATE STOPPING PLACES WHICH THE CARRIER DEEMS APPROPRIATE. THE SHIPPER'S ATTENTION IS DRAWN TO THE NOTICE CONCERNING CARRIER'S LIMITATION OF LIABILITY. Shipper may increase such limitation of liability by declaring a higher value for carriage and paying a supplemental charge if required.",
       "shipper": {
-        "type": "Organization",
+        "type": [
+          "Entity"
```
Comment (OR13) [inline docs/openapi/components/schemas/credentials/IATAAirWaybillCertificate.yml:90]: change to Organization

**Review by OR13 — CHANGES_REQUESTED** [review]
don't use "Entity"

File: docs/openapi/components/schemas/common/BillOfLading.yml [hunk 2]
```diff
@@ -174,28 +174,28 @@ example: |-
     ],
     "scac": "NISC",
     "carrier": {
-      "type": "Entity",
+      "type": ["Entity"],
```
Comment (OR13) [inline docs/openapi/components/schemas/common/BillOfLading.yml:177]: same comment

**Review by mprorock — APPROVED** [review]

**Comment by nissimsan** [comment]
Merging out of call when changed back to Org

**Comment by nissimsan** [comment]
@OR13, doubled down changing Organization to Entity everywhere now. Merging.

**Comment by TallTed** [comment]
> [@nissimsan] @OR13, doubled down changing Organization to Entity everywhere now. Merging.

Looking at a couple of the latest commits, I'm pretty sure that should have said ```changing Entity to Organization```

**Comment by nissimsan** [comment]
@TallTed, aargh - yes, absolutely!! 🤦‍♂️

**Pull request merged**